src/iOS/toga_iOS/widgets/image.py

```diff
@@ -1,12 +1,24 @@
-from toga_iOS.libs import UIImage
+import os
+from toga_iOS.libs import(
+    NSData,
+    NSURL,
+    UIImage
+)
 
 
 class Image(object):
     def __init__(self, interface):
         self.interface = interface
 
     def load_image(self, path):
+        # If a remote URL is provided, use the download from NSData (similar to toga-cocoa)
         if path.startswith('http://') or path.startswith('https://'):
             self.native = UIImage.imageWithData_(NSData.dataWithContentsOfURL_(NSURL.URLWithString_(path)))
         else:
-            self.native = UIImage.alloc().initWithContentsOfFile_(path)
+            basename = os.path.basename(path)
+            if basename == path:
```

Comment:
Can this differentiate between a named image, and an image that is in the root of the resources directory?

Reply:
I've removed imageNamed initialization for now, as it goes against normal usage of toga widgets - I was thinking only of iOS when I first implemented it.